### ISSUE TITLE
Does the current bar chart supports stacked column?

### ISSUE THREAD

**Issue by joeylu** [capitalone/react-native-pathjs-charts#106]
would be nice if we have this :)

**Comment by marzolfb** [comment]
Agreed it would be nice to have this. I looked at this a bit and creation of the bar chart is highly dependent on the [paths-js library](https://github.com/andreaferretti/paths-js) and this library does not have support for stacked columns. So, either we'd need to see if we could get that added into that lib via a PR or on another extreme consider a rewrite that doesn't depend on paths-js.
